packages/front-end/components/Features/Rule.tsx

```diff
@@ -60,6 +64,41 @@ export const Rule = forwardRef<HTMLDivElement, RuleProps>(
       permissions.check("manageFeatures", feature.project) &&
       permissions.check("createFeatureDrafts", feature.project);
 
+    const hasConditions = rule.condition && rule.condition !== "{}";
+
+    let upcomingScheduleRule: ScheduleRule | null = null;
```

Comment:
This is done in two places in the app, I should explore making this a service function.